x-pack/plugins/alerting/server/saved_objects/mappings.json

```diff
@@ -163,6 +163,16 @@
                 "type": "keyword"
               }
             }
+          },
+          "warning": {
```

Comment:
@XavierM fyi, we are adding warning status in this PR. Did you want someone from R&AM to review?